doc/source/cluster/vms/user-guides/community/spark.rst

```diff
@@ -17,29 +17,31 @@ Assuming the python file name is 'ray-on-spark-example1.py'.
 .. code-block:: python
 
     from pyspark.sql import SparkSession
-    from ray.util.spark import init_ray_cluster, shutdown_ray_cluster, MAX_NUM_WORKER_NODES
+    from ray.util.spark import setup_ray_cluster, shutdown_ray_cluster, MAX_NUM_WORKER_NODES
```

Comment:
Our spark users prefers the name setup_ray_cluster, because we already have  a spark cluster, if we say start_ray_cluster it sounds like start another cluster independent with the spark cluster.